src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -2244,7 +2244,7 @@ class PvtModelTableParametersDescription:
     warn_when_outside: bool = attr.ib(default=True, validator=instance_of(bool))
 
     table_type: Optional[str] = attr.ib(
-        default=None, validator=optional(instance_of(str))
+        default=constants.PVTTableType.PT.value, validator=optional(instance_of(str))
```

Comment:
Is this `.value` required?

couldn't it be just `default=constants.PVTTableType.PT, ...`?

Reply:
I could try to change it to be an `Enum`. In the first `enthalpy` PR I tried to do it but I had some problems with the type checking for the `Enum`.

Also, this `.table_type` attribute is being passed to the `.mode` attribute that needs to be a `str`. Perhaps the `PvtModel` `mode` attribute could also be changed to an `Enum`. 

This change will impact in other places, since now this is being used also to import the PH tables. I could open a PR only to it if you all agree?

Reply:
No rush. perhaps create a task to tackle this and do it when you think it is more appropriate.

Reply:
I added this point into the task [ASIM-4887](https://eden.esss.co/jira/browse/ASIM-4887) description. So I could remember to do it latter.

Reply:
Did you use `attrib_enum` similar to this example?
https://github.com/ESSS/alfasim-sdk/blob/565119cbbfbad72e71a34bac5dbcc6ed91ed19d3/src/alfasim_sdk/_internal/alfacase/case_description.py#L94-L97

Reply:
@Gualcar could you check if @prusse-martin's suggestion solves the problem? I

Reply:
Thank you for the tip @prusse-martin, I tried using this `attrib_enum` and I had problems with type checking but I not investigated it a lot.

Probably I used it wrong, because there is a difference passing `constants.PVTTableType.PT` (type `PVTTableType`), `constants.PVTTableType` (type `EnumMeta`) or `constants.PVTTableType.PT.value` (type `str`).

I will try it.

Reply:
Please see if you can avoid using the `.value` and use `attrib_enum` as @prusse-martin suggested. If you run into type-check problems, call any of us for help.

It is better to avoid temporary workarounds in `alfasim-sdk` since this is an open source project and any change here may affect external users of the sdk.

Give it a try again, it seems like a simple change. These type-checking errors may be a simple detail that we're missing, a simple call may solve it ;)

Reply:
As we discussed in the meeting today we will divide the implementations of the PT and PH table. This way there will be a `TableDescription` for each table and the `table_type` parameter will not be required.

This refactoring will affect different regions of the code and is outside this PR scope. So I will finish it and latter I will a do this refactoring (issue ASIM-4887).

Reply:
when using `attrib_enum` the schema used to validate the input file will check the valid values, when using a string no special type check is done.